src/pages/affiliates/AffiliatesPage.tsx

```diff
@@ -122,6 +123,8 @@ export const AffiliatesPage = () => {
         <$ContentAttached>
           <Outlet context={{ accountStats, programStats }} />
         </$ContentAttached>
+
+        <AffiliatesLeaderboard {...{ accountStats }} />
```

Comment:
what is the reason for not using the outlet?

Reply:
We've added the leaderboard on both tabs, so it's rendered in affiliates page without the tabs logic.